src/features/pass-survey/model/pipelineBuilder.ts

```diff
@@ -234,7 +234,17 @@ export function buildPipeline(activity: ActivityDetails): PipelineItem[] {
     })
     .reduce<PipelineItem[]>((items, item) => {
       return Array.isArray(item) ? [...items, ...item] : [...items, item];
-    }, []);
+    }, [])
+    .map((item, index) => {
+      return {
+        ...item,
+        isAbleToMoveBack:
+          !activity.responseIsEditable && index !== 0
```

Comment:
Please remove or modify the `index !== 0` condition because we also need to take into account that the item is not "Splash".
Also, I would recommend moving the complex condition into a separate variable.